python/lsst/ts/FiberSpectrograph/fiberSpectrograph.py

```diff
@@ -271,14 +271,23 @@ def _configure_ctypes(self):
 
         Some of the functions in libavs need to have their types and/or
         return values explicitly defined as pointers for python to be able
-        to pass them in correctly.
+        to pass them in correctly using the `ctypes` interface. Any C function
+        that manipulates a ctypes pointer should have its argument types
+        defined here.
```

Comment:
I thought you told me it was "ctypes pointer to something other than a plain old data type".

Reply:
I don't know that for certain, so I'm just going to be careful and explicitly define it for every method that takes a pointer.